get_mock_server.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-if ! [ -e $1/mock_server-x64-linux ]; then
-  curl -L -o $1/mock_server-x64-linux https://github.com/googleinterns/cloud-operations-api-mock/raw/master/cmd/mock_server-x64-linux
-  chmod +x $1/mock_server-x64-linux
+if ! [ -e $1/mock_server-x64-linux-v1-alpha ]; then
+  curl -L -o $1/mock_server-x64-linux-v1-alpha https://github.com/googleinterns/cloud-operations-api-mock/releases/download/v1-alpha/mock_server-x64-linux-v1-alpha
+  chmod +x $1/mock_server-x64-linux-v1-alpha
```

Comment:
- Can you create a variable for version string to use in the URL?
- Would be clean if you create a symbolic link from the full name version binary to a constant alias like `mock_server`. That way, in the actual test code it can just call `mock_server` and not need updating with version changes

Reply:
Done, symlink created.